app/assets/stylesheets/markdown-editor.scss

```diff
@@ -27,6 +27,7 @@
 
   .btn-group {
     margin-bottom: 8px;
+    &:first-child { margin-left: 0; }
```

Comment:
It looks to me like there is no margin set anywhere else, so this line looks unnecessary to me.